web_external/templates/body/dataPanel.jade

```diff
@@ -1,30 +1,25 @@
-.dataPanelContainer
-  .panelTitle
-    | Available Datasets
-  .savedDatasets
-    .savedDatasetsTitle
-    .savedDataList
-        .datasets
-            each dataset in datasets
-                if dataset.metadata()
-                    .dataset
-                      span(title=dataset.get('name'))= dataset.get('name').length > 13 ? dataset.get('name').slice(0,13) + "..." : dataset.get('name')
-                        - var attributes = {'m-dataset-id': dataset.get('_id')}
-                        if dataset.isRenderable()
-                            //- Eye icon to move dataset into current session
-                            - var classes = (dataset.get('displayed') ? 'icon-eye icon-disabled dataset-in-session' : 'icon-eye icon-enabled add-dataset-to-session')
-                            i(class=classes)&attributes(attributes)
-                        //- trash icon to delete dataset
-                        - var classes = (dataset.get('displayed') ? 'icon-trash delete-dataset icon-disabled dataset-in-session' : 'icon-trash delete-dataset icon-enabled')
-                        i(class=classes)&attributes(attributes)
-                        //- info icon for minerva metadata display
-                        i.icon-info-circled.icon-enabled.dataset-info&attributes(attributes)
-                        if dataset.get('meta').minerva.original_type === 'csv' || dataset.get('meta').minerva.original_type === 'json' || (dataset.get('meta').minerva.original_type === 'mongo' && !('spatial_field' in dataset.get('meta').minerva))
-                            //- table icon for mapping columns of csv/json
-                            - var displayed = dataset.get('displayed')
-                            if !displayed && (dataset.get('meta').minerva.original_type === 'json' || dataset.get('meta').minerva.original_type === 'mongo') && (!dataset.get('meta').minerva.json_row)
-                                //- when there is no json data, disable the table icon, as if the dataset was displayed
-                                - displayed = true
-                            - var classes = (displayed ? 'icon-table csv-mapping icon-disabled dataset-in-session' : 'icon-table csv-mapping icon-enabled')
-                            i(class=classes)&attributes(attributes)
+include ../layout/panelMixins.jade
 
++panel-title('Available Datasets')
```

Comment:
Is it possible to put more of this repeated logic in the jade mixin for panel-title?  i.e. passing `#m-data-panel` is the only thing different between here and the jobs panel.  Though maybe this would complicate matters when adding additional icons like in the sources panel?

At the very least, please document this standard jade for a panel in the Extending Minerva doc.